rpc/wrtc_client_channel.go

```diff
@@ -336,29 +333,42 @@ func (ch *webrtcClientChannel) writeReset(stream *webrtcpb.Stream) error {
 
 // taken from
 // https://github.com/grpc-ecosystem/go-grpc-middleware/blob/560829fc74fcf9a69b7ab01d484f8b8961dc734b/logging/zap/client_interceptors.go
-
-func logFinalClientLine(ctx context.Context, startTime time.Time, err error, msg string) {
+func logFinalClientLine(logger utils.ZapCompatibleLogger, startTime time.Time, err error, msg string) {
 	code := grpc_logging.DefaultErrorToCode(err)
 	level := grpc_zap.DefaultCodeToLevel(code)
+
+	// this calculation is done because duration.Milliseconds() will return an integer, which is not precise enough.
+	duration := float32(time.Since(startTime).Nanoseconds()/1000) / 1000
```

Comment:
Seems fair, but why did you feel we needed the extra precision?

Reply:
a lot of calls take less than 1ms, so would just show up as 0ms. the math here ends up giving 3 decimal places, which is pretty good